src/main_thread/init/multi_thread_content_initializer.ts

```diff
@@ -1779,6 +1782,15 @@ export default class MultiThreadContentInitializer extends ContentInitializer {
                 resetMediaElement(mediaElement, url);
               });
               mediaSourceStatus.setValue(MediaSourceInitializationStatus.Attached);
+
+              if (isManagedMediaSource) {
+                /**
+                 * Using ManagedMediaSource needs to disableRemotePlayback or to provide
+                 * an Airplay source alternative, such as HLS.
+                 * https://github.com/w3c/media-source/issues/320
+                 */
+                mediaElement.disableRemotePlayback = true;
```

Comment:
Maybe we should restore the previous value when not relying on the media element anymore?

Reply:
How do we know the previous value ?

Reply:
By storing it somewhere I guess, or is there some detail I miss?